job_bundle_output_tests/physical/scene/cube.py

```diff
@@ -1,13 +1,14 @@
 import os
+
```

Comment:
Huh, there's a specific rule that says the c4d import comes at the end after a newline?

Reply:
Yeah, according to [the PEP8 Python style guide](https://peps.python.org/pep-0008/#imports):

> Imports should be grouped in the following order:
> 1 .Standard library imports.
> 2. Related third party imports.
> 3. Local application/library specific imports.